apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -336,19 +336,19 @@ const DanceAiModal: React.FunctionComponent<DanceAiProps> = ({onClose}) => {
           {mode === Mode.SELECT_INPUTS
             ? 'Choose three emoji for the mood of the effects.'
             : mode === Mode.PROCESSING && !processingDone
-            ? 'The AI is processing your input.'
+            ? 'A.I. is processing your input.'
```

Comment:
I noticed that currently in instructions, 'AI' and not 'A.I.' is being used. Adding periods will be used everywhere then for the abbreviation?

Reply:
Also in discussion with @samantha-code we decided to try referring to the bot, who is A.I., for this revision.